docs/csharp/language-reference/keywords/out-parameter-modifier.md

```diff
@@ -53,12 +64,29 @@ The `out` keyword causes arguments to be passed by reference. This is like the [
 -   Async methods, which you define by using the [async](../../../csharp/language-reference/keywords/async.md) modifier.  
   
 -   Iterator methods, which include a [yield return](../../../csharp/language-reference/keywords/yield.md) or `yield break` statement.  
-  
-## Example  
+
+## Declaring an `out` method   
```

Comment:
It's not really an `out` method. It's a method with `out` arguments. Maybe 

## Declaring `out` arguments

is a better heading?